install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -419,3 +422,19 @@ see what caused the errors:
 $ openstack stack failures list openshift-cluster
 ----
 ====
+
+
+=== Stack Name Configuration
+
+By default the Heat stack created by OpenStack for the OpenShift cluster will
```

Comment:
By default,
s/created/that is created
s/OpenShift/{product-title}
s/will be/is